NorthstarDedicatedTest/rpakfilesystem.cpp

```diff
@@ -165,7 +165,7 @@ int LoadPakAsyncHook(char* path, void* unknownSingleton, int flags, void* callba
 		LoadPreloadPaks();
 		LoadCustomMapPaks(&path, &bNeedToFreePakName);
 
-		bShouldLoadPaks = true;
+		// bShouldLoadPaks = true; // why are we ever loading these more than once? i dont understand why we were setting this to true again
```

Comment:
wouldn't this break mod pak loading entirely after like, literally the first one the game loads?

Reply:
we set bShouldLoadPaks to false at all to prevent recursive LoadPak calls, but it needs to be set back to true afterwards so subsequent calls will still try to load mod paks

Reply:
ah ok, wasnt sure how non-preload paks were loaded, will fix